cbbpoll/templates/index.html

```diff
@@ -3,32 +3,71 @@
 {% block content %}
 <div class="jumbotron">
     {% if not g.user.is_authenticated -%}
-        <h1>Welcome All!</h1>
+    <h1>Welcome All!</h1>
     {% elif not g.user.is_admin() %}
-        <h1>Howdy, partner!</h1>
+    <h1>Howdy, partner!</h1>
     {% else%}
-        <h1>"Awaiting instructions..."</h1>
+    <h1>"Awaiting instructions..."</h1>
     {%- endif %}
-{% if not g.user.is_authenticated -%}  
-  <p>This is the site for the <a href='//reddit.com/r/CollegeBasketball'>/r/CollegeBasketball</a> User Poll. Applications for voters are currently being accepted. Please log in to apply or to submit a ballot.</p>
-  <p><a href="{{ url_for('login') }}" class="btn btn-primary btn-lg">Login Using Reddit</a></p>
-{% elif open_poll -%}
-  <p>This is the site for the <a href='//reddit.com/r/CollegeBasketball'>/r/CollegeBasketball</a> User Poll. The {{open_poll}} is currently open for ballot submission until {{timestamp(closes_eastern)}}. Applications for voters are currently being accepted!</p>
-  <p>{% set ballot = g.user.submitted_ballot_to(open_poll.id) %}{% if ballot %}<a href="{{url_for('ballot', ballot_id = ballot.id)}}" class="btn btn-primary btn-lg">View Your Ballot{% else %}<a href="{{ url_for('submitballot') }}" class="btn btn-primary btn-lg">Submit Your Ballot{%endif%}</a></p>
-{% elif not poll -%}
-  <p>This is the site for the <a href='//reddit.com/r/CollegeBasketball'>/r/CollegeBasketball</a> User Poll. There are no completed polls yet, but we can't wait for the season to start! Applications for voters are currently being accepted.</p>
-{% else -%}
-  <p>This is the site for the <a href='//reddit.com/r/CollegeBasketball'>/r/CollegeBasketball</a> User Poll. There are currently no polls open for ballot submission.</p>
-{%- endif %}
-{% if g.user.is_authenticated -%}
-<a href="{{url_for('apply')}}" class="btn btn-success btn-lg">Apply for Voter Status</a>
-{% endif %}
-</div>
-{% if poll %}
-<h2> Latest Poll Results </h2>
-<div class="row">
-<div class="h6 col-xs-8 col-md-5">Closed on {{timestamp(closed_eastern)}}</div><div class="h6 col-xs-4 col-md-3 text-right"><a href="{{url_for("polls", s=poll.season|int, w= poll.week|int)}}">Permalink to these Results</a></div>
+    {% if not g.user.is_authenticated -%}
+    <p>Welcome to the Reddit College Basketball Tournament Challenge!</p>
```

Comment:
This element is exactly the same regardless of user authentication level - should be moved outside the conditional blocks to avoid repeating yourself.